options/locale/locale_zh-TW.ini

```diff
@@ -1855,7 +1855,7 @@ settings.add_telegram_hook_desc=將 <a href="%s">Telegram</a> 整合到您的儲
 settings.add_matrix_hook_desc=將 <a href="%s">Matrix</a> 整合到您的儲存庫。
 settings.add_msteams_hook_desc=將 <a href="%s">Microsoft Teams</a> 整合到您的儲存庫。
 settings.add_feishu_hook_desc=將 <a href="%s">Feishu</a> 整合到您的儲存庫。
-settings.add_Wechat_hook_desc=將 <a href="%s">Wechatwork</a> 整合到您的儲存庫。
+settings.add_wechatwork_hook_desc=將 <a href="%s">Wechatwork</a> 整合到您的儲存庫。
```

Comment:
non-english translations will be overwritten by the CI. Per @zeripath's suggestion, changing the template may be a better approach.

Reply:
Okay, fixed it.